### PR TITLE
[Triton] Update recipe to build TRT engine and model_repo for Conformer offline model

### PR DESCRIPTION
**Pull request by wd929**
Hi all,

This PR updates recipes to build TRT engine and a unified model_repo for Conformer offline model.
I added additional TRT building script using same model_repo structure as ONNX model with several parameters changing.
FP16 ONNX model can be used when set fp16=true, where we use _polygraphy convert_ to perform the FP32->FP16
I added one audio feature for TRT inference testing.
Also updated the README for Triton.

Thanks!

### CODE REVIEW

File: triton/scripts/build_librispeech_pruned_transducer_stateless3_offline.sh [hunk 1]
```diff
@@ -58,15 +58,11 @@ if [ ${stage} -le -1 ] && [ ${stop_stage} -ge -1 ]; then
     echo "export onnx"
     cp scripts/*onnx*.py ${recipe_dir}/
```
Comment (yuekaizhang) [inline triton/scripts/build_librispeech_pruned_transducer_stateless3_offline.sh:59]: We may not need this cp anymore, if we are trying use the icefall native export-onnx.py.

File: triton/scripts/build_librispeech_pruned_transducer_stateless3_offline.sh [hunk 2]
```diff
@@ -108,9 +104,9 @@ if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
 fi
 
 if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
-    cp $pretrained_model_dir/exp/encoder_fp16.onnx $model_repo_path/encoder/1/encoder.onnx
-    cp $pretrained_model_dir/exp/decoder_fp16.onnx $model_repo_path/decoder/1/decoder.onnx
-    cp $pretrained_model_dir/exp/joiner_fp16.onnx $model_repo_path/joiner/1/joiner.onnx
+    cp $pretrained_model_dir/exp/encoder-epoch-9999-avg-1.onnx $model_repo_path/encoder/1/encoder.onnx
```
Comment (yuekaizhang) [inline triton/scripts/build_librispeech_pruned_transducer_stateless3_offline.sh:107]: Also, could we try use polygraphy or onnxmltools code to generate  fp16 onnx files? Then we could make the onnx recipe using fp16.

Actually, I have not tried use  $pretrained_model_dir/exp/encoder-epoch-9999-avg-1.int8.onnx. If it works with triton, we might could choose one solution between int8 and fp16 based on throughput.

File: triton/scripts/build_librispeech_pruned_transducer_stateless3_offline.sh (hunk 1 above)
Comment (wd929) [reply triton/scripts/build_librispeech_pruned_transducer_stateless3_offline.sh:59]: Yeah sure!

File: triton/scripts/build_librispeech_pruned_transducer_stateless3_offline.sh (hunk 2 above)
Comment (wd929) [reply triton/scripts/build_librispeech_pruned_transducer_stateless3_offline.sh:107]: Currently, the native icefall/export-onnx.py will not generate the FP16 model so I keep FP32 here. 
But I can test with int8 to see whether these models can be used in Triton. Will keep you updated

Comment (yuekaizhang) [reply triton/scripts/build_librispeech_pruned_transducer_stateless3_offline.sh:107]: Thanks. Btw, "polygraphy convert --fp-to-fp16 -o encoder_fp16.onnx encoder-epoch-9999-avg-1.onnx" will be helpful.

**Comment by csukuangfj** [comment]
@yuekaizhang 

Thanks for reviewing. Please merge it if you think it is ready.

**Review by yuekaizhang — APPROVED** [review]

**Comment by yuekaizhang** [comment]
Many thanks! Merge it.

**Pull request merged**